pkg/bpf/workload/xdp.go

```diff
@@ -98,6 +98,27 @@ func (xa *BpfXdpAuthWorkload) LoadXdpAuth() error {
 	xa.Info.Type = prog.Type
 	xa.Info.AttachType = prog.AttachType
 
+	if err = xa.XdpTailcallMap.Update(
+		uint32(constants.TailCallPolicyCheck),
+		uint32(xa.PolicyCheck.FD()),
+		ebpf.UpdateAny); err != nil {
+		return err
```

Comment:
In the case where only error is returned, is it possible to know if it is a policyCheck error or a ruleCheck error?